smk-edit/static/store/config-tool-geomark.js

```diff
@@ -0,0 +1,17 @@
+export default {
+    filters: {
+        'geomark': {
+            get: function ( tool ) {
+                return Object.assign( {
+                    title: 'Geomark',
```

Comment:
Should we be specifying a default order and position here like we do for some of the other tools? Or is it unnecessary? 
I'm just thinking it might look better in the UI when values are pre-populated.

Reply:
I don't know of a reason to specify a default order or position for Geomark, and I'm not sure why some tools do specify a default. Values can be set for these when editing the Geomark tool.

Reply:
My thinking is that the less configuration a user feels they have to do the better. If there is a blank space, the user feels compelled to populate it with something and it's not immediately obvious what they would enter for the 'order' field anyhow.

Reply:
One more argument for providing defaults...consistency...I think all the other panel tools provide defaults!

Reply:
Thanks. It does look like nearly all panels have a default order, so I'll add one. I did a quick survey of existing order values for panel tools:

about 1
base maps 3
layers 3
route planner/directions 4
identify 5
version (no value)

I'll go with 3 as it's used most often, and results in Geomarks appearing in the middle and alphabetically ordered with other panels having an order of 3.